Gwayeon/Models/User.swift

```diff
@@ -12,10 +12,12 @@ struct User: Codable {
     var uid: String
     var username: String
     var friends: [String]
+    var recommends: [String]
```

Comment:
```suggestion
    var uid: String
    var username: String
    var friends: [String]?
    var recommends: [String]?
```
p1;
친구와 추천이 없을 때를 고려해야하지 않을까여?

Reply:
https://github.com/DeveloperAcademy-POSTECH/MC3_Team5_NealAndBori/pull/99/commits/2164788d6b941d03c9dd6462e927d396e846021e 반영했습니다~